tests/test_point/test_point_arithmetic.py

```diff
@@ -0,0 +1,106 @@
+import contextlib
+
+import pytest
+from cryptography.hazmat.backends.openssl import backend
+from cryptography.hazmat.primitives.asymmetric import ec
+
+from umbral.bignum import BigNum
+from umbral.config import default_curve
+from umbral.point import Point
+
+curve = default_curve() or ec.SECP256K1()
+
+
+@pytest.fixture()
+def mock_openssl(mocker, random_ec_point1, random_ec_bignum):
+    @contextlib.contextmanager
+    def mocked_openssl_backend():
+        def mocked_ec_point_equality(group, ec_point, other_point, bn_ctx):
+            assert 'BN_CTX' in str(bn_ctx)
+            assert 'EC_GROUP' in str(group)
+            assert 'EC_POINT' in str(ec_point)
+            assert 'EC_POINT' in str(other_point)
+
+            assert random_ec_point1.group == group
+            assert random_ec_point1.ec_point == ec_point
+            assert random_ec_point1.ec_point == other_point
+
+            return 1
```

Comment:
Perhaps a comment here, explaining that this is the expected output of the mocked function(s)?